doc/api/cluster.markdown

```diff
@@ -357,18 +357,19 @@ This example will echo back all messages from the master:
       });
     }
 
-### worker.suicide
+### worker.exitedVoluntarily (formerly worker.suicide)
 
 * {Boolean}
 
 Set by calling `.kill()` or `.disconnect()`, until then it is `undefined`.
```

Comment:
@sam-github I don't think that removing in 7.x is a viable option here, given that it's used and documented as being «stable». It should be kept for longer time as being soft-deprecated (at least one major release), then for some time as being hard-deprecated before actual removal.